packages/console/src/components/PlanUsage/index.tsx

```diff
@@ -59,6 +59,9 @@ function PlanUsage({ periodicUsage: rawPeriodicUsage }: Props) {
       usage:
         key === 'mauLimit' || key === 'tokenLimit'
           ? periodicUsage[key]
```

Comment:
This expression is too complicated. Need to split into several smaller pieces

Reply:
Refactored.